src/main/webapp/app/complaints/form/complaints-form.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { ComplaintService } from 'app/complaints/complaint.service';
+import { Result } from 'app/entities/result.model';
+import { HttpErrorResponse } from '@angular/common/http';
+import { Complaint, ComplaintType } from 'app/entities/complaint.model';
+import { Exercise } from 'app/entities/exercise.model';
+import { onError } from 'app/shared/util/global.utils';
+import { AlertService } from 'app/core/util/alert.service';
+
+@Component({
+    selector: 'jhi-complaint-form',
+    templateUrl: './complaints-form.component.html',
+    styleUrls: ['complaints-form.component.scss'],
```

Comment:
as you did it for the templateUrl as well, you could integrate the styleUrl in the same manner: `./complaints-form.component.scss`